CP04OSSM/CP04OSSM_R00001_ingest.csv

```diff
@@ -37,6 +37,8 @@ mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP04
 #mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl11/mopak_*/*.mopak.log,CP04OSSM-SBD11-01-MOPAK0000,recovered_host,Not Expected,no recovered instrument data
 #mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl11/Hydrogen_sn/*.hyd1.log,CP04OSSM-SBD11-02-HYDGN0000,recovered_host,Not Expected,no recovered instrument data
 #mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl11/METBK-Log_*/*.metbk*.log ,CP04OSSM-SBD11-06-METBKA000,recovered_host,Not Expected,no recovered instrument data
+#mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl11/METBK-Log_*/*.metbk*.log ,CP04OSSM-SBD11-06-METBKA001,recovered_host,Not Expected,no recovered instrument data
+#mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00001/cg_data/dcl11/METBK-CT_sn11001/*.hex,CP04OSSM-SBD11-06-METBKA001,recovered_inst,Not Expected,no recovered instrument data

```

Comment:
same comments/questions as with early ISSM deployments - no rec_instr folder on raw data repo, so I think the 'mapping' above is ok (METBK-Log instead of METBK-CT on line 40??).  I think METBK-CT* should be fine instead of listing SN on line 41?